<launch>
    <arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
<!--    <arg name="launch_prefix" default="gdb -ex run &#45;&#45;args" /> &lt;!&ndash; set to value="gdbserver localhost:10000" for remote debugging &ndash;&gt;-->
    <arg name="apriltag_localization_name" default="apriltag_localization_with_camera_node" />

    <arg name="cam_config"  default="$(find apriltag_localization)/config/"/>
    <arg name="odom_topic"  default="/jarvis/odom"/>

    <include file="$(find apriltag_localization)/launch/apriltag_localization_with_camera.launch">
<!--    <include file="$(find apriltag_localization)/launch/apriltag_localization_nodelet.launch">-->
        <arg name="launch_prefix" value="$(arg launch_prefix)" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
        <arg name="apriltag_localization_name" value="$(arg apriltag_localization_name)" />
        <arg name="cam_config"  value="$(arg cam_config)"/>
        <arg name="odom_topic"  value="$(arg odom_topic)"/>
    </include>

    <node pkg="apriltag_localization" type="fake_odom_node" name="fake_odom_node" output="screen" />

    <node pkg="rostopic" type="rostopic" name="start_pub" args="pub /tag_cam_name std_msgs/String 'd455@bundle5' "  />
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_view"/>

</launch>